sympy/utilities/misc.py

```diff
@@ -44,7 +44,7 @@ def rawlines(s):
     >>> from sympy.utilities.misc import rawlines
     >>> from sympy import TableForm
     >>> s = str(TableForm([[1, 10]], headings=(None, ['a', 'bee'])))
-    >>> print(rawlines(s)) # the \\ appears as \ when printed
```

Comment:
This comment was referring to the way the file looks, not the rendered docstring (when viewed from `help()` it has no `\\`). Unfortunately, this docstring cannot be made raw, since it contains both `"""` and `'''` as substrings, so I've left it as-is in its harder-to-read-in-source form.